src/BrandedNavBar/SubMenuTrigger.tsx

```diff
@@ -57,9 +34,8 @@ const SubMenuTriggerButton = React.forwardRef<
   <StyledButton isOpen={isOpen} ref={ref} {...props}>
     {name}
     <Icon
-      style={{ position: "absolute", top: "7px" }}
+      style={{ position: "absolute", top: "10px" }}
```

Comment:
These values don't seem to be based on our 8px grid system, is that ok?

Reply:
@haideralsh unfortunately the size of the chevron Icon is not the size of the box that it's in, so we had to do custom positioning to put it at the centre of the line height.